nmdc_schema/migrators/migrator_from_11_1_0_to_11_2_0.py

```diff
@@ -0,0 +1,71 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+
+
+class Migrator(MigratorBase):
+    r"""Migrates a database between two schemas."""
+
+    _from_version = "11.1.0"
+    _to_version = "11.2.0"
+
+    def upgrade(self) -> None:
+        r"""
+        Migrates the database from conforming to the original schema, to conforming to the new schema.
+
+        >>> from nmdc_schema.migrators.adapters.dictionary_adapter import DictionaryAdapter
+        >>> database = dict(
+        ...   data_generation_set=[
+        ...       {'id': 1, 'type': 'nmdc:MassSpectrometry'},
+        ...       {'id': 2, 'type': 'nmdc:MassSpectrometry', 'has_calibration': 'nmdc:calib-99-abc123'},
+        ...       {'id': 3, 'type': 'nmdc:NucleotideSequencing', 'has_calibration': 'nmdc:calib-99-abc123'},
```

Comment:
This is not a valid class for has_calibration for schema 11.1.0

Reply:
Thanks for bringing this to my attention. I'll remove this example.